data/src/test/java/org/apache/iceberg/data/DataTest.java

```diff
@@ -40,6 +41,7 @@
 import org.apache.iceberg.types.Types.StructType;
 import org.apache.iceberg.util.DateTimeUtil;
 import org.assertj.core.api.Assumptions;
+import org.junit.Assert;
```

Comment:
Please use Junit5 in tests.
We are still migrating, but junit5 is the goal
Please avoid using junit 4 in tests.